trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/system/table/GravitinoSystemTableFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.trino.connector.system.table;
+
+import com.datastrato.gravitino.trino.connector.catalog.CatalogConnectorManager;
+import com.google.common.base.Preconditions;
+import io.trino.spi.Page;
+import io.trino.spi.connector.ConnectorTableMetadata;
+import io.trino.spi.connector.SchemaTableName;
+import java.util.HashMap;
+import java.util.Map;
+
+/* This class managed all the system tables */
+public class GravitinoSystemTableFactory {
+
+  private final CatalogConnectorManager catalogConnectorManager;
+  public static Map<SchemaTableName, GravitinoSystemTable> systemTableDefines = new HashMap<>();
```

Comment:
If the reference of `systemTableDefines` will not be changed, I would suggest adding a `final` flag.